src/web/theme/modules/Header/Navigation/Menu/Menu.scss

```diff
@@ -0,0 +1,94 @@
+.menu {
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  top: 0;
+  transition: transform 0.2s ease-in-out;
+  z-index: 1;
+
+  &.close {
+    transform: translateX(calc(-100% + 3em));
+
+    button.menu__button.button {
+      background-color: $backgroundColor;
+      padding: 1em 1em;
+    }
+    .menu__navButton {
+      flex: 1;
+    }
+  }
+
+  &.open {
+    button.menu__button.button {
```

Comment:
`menu__button` should be enough here. If not, there might be an issue with another css rule.